sfa_dash/blueprints/reports.py

```diff
@@ -279,7 +279,7 @@ def get(self, uuid):
         except DataRequestException as e:
             self.flash_api_errors(e.errors)
         else:
-            flash('Report recomputed successfully.', 'message')
+            flash('Scheduled report recompute successfully.', 'message')
```

Comment:
Should we add text along the lines of wait 30 seconds and refresh the page? Or add a timer? (like we have for the original waiting page)

Reply:
The user should still land back on that page and be able to see that message. I believe there is also a script Tony added to automagically reload the page on a timer.

Reply:
I'm confused. This just adds a box to the previously rendered report. It doesn't send the user to another page.

Reply:
Ah, I was thinking that the report status was reset to pending on recompute. Now I see that it continues to serve the old report until it gets recomputed. Updated the text.